src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -366,13 +366,13 @@ void ControlCompOutput(EnergyPlusData &state,
                     ZoneInterHalf.MaxResult = 1.0;
                     ZoneInterHalf.MinResult = 0.0;
                     // If between the max and mid set to new flow and raise min to mid
-                } else if ((ZoneController.SetPoint < ZoneInterHalf.MaxResult) && (ZoneController.SetPoint >= ZoneInterHalf.MidResult)) {
+                } else if (ZoneController.SetPoint >= ZoneInterHalf.MidResult) {
                     ZoneController.CalculatedSetPoint = (ZoneInterHalf.MaxFlow + ZoneInterHalf.MidFlow) / 2.0;
                     ZoneInterHalf.MinFlow = ZoneInterHalf.MidFlow;
                     ZoneInterHalf.MinResult = ZoneInterHalf.MidResult;
                     ZoneInterHalf.MidFlow = (ZoneInterHalf.MaxFlow + ZoneInterHalf.MidFlow) / 2.0;
                     // If between the min and mid set to new flow and lower Max to mid
-                } else if ((ZoneController.SetPoint < ZoneInterHalf.MidResult) && (ZoneController.SetPoint > ZoneInterHalf.MinResult)) {
+                } else {
```

Comment:
[src/EnergyPlus/GeneralRoutines.cc:375]:(style),[knownConditionTrueFalse],Condition 'ZoneController.SetPoint>ZoneInterHalf.MinResult' is always true

Reply:
Reducing these on the fly is a little scary but these changes do seem correct.